<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CartWindow</class>
 <widget class="QMainWindow" name="CartWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>534</width>
    <height>790</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lbCartTitle">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ваша корзина</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>450</width>
      <height>620</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>448</width>
       <height>618</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QWidget" name="widget" native="true"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnBuy">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>710</y>
      <width>181</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Купить</string>
    </property>
   </widget>
   <widget class="QLabel" name="productsInCartLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Товары в корзине</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBack">
    <property name="geometry">
     <rect>
      <x>362</x>
      <y>40</y>
      <width>131</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Вернутся назад</string>
    </property>
   </widget>
   <widget class="QLabel" name="totalPriceLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>710</y>
      <width>261</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Всего: </string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>534</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action">
   <property name="text">
    <string>Назад </string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
